src/Packetery/Module/Order/LabelPrint.php

```diff
@@ -220,16 +223,25 @@ public function outputLabelsPdf(): void {
 		$isCarrierLabels        = ( $this->httpRequest->getQuery( self::LABEL_TYPE_PARAM ) === self::ACTION_CARRIER_LABELS );
 		$idParam                = $this->httpRequest->getQuery( 'id' );
 		$packetIdParam          = $this->httpRequest->getQuery( 'packet_id' );
+		$orders                 = [];
+		$packetIds              = [];
 		if ( null !== $idParam && null !== $packetIdParam ) {
 			$fallbackToPacketaLabel = true;
-			$packetIds              = [ $idParam => $packetIdParam ];
+			try {
+				$orders[ $idParam ] = $this->orderRepository->getById( (int) $idParam );
+				$packetIds          = [ $idParam => $packetIdParam ];
+				// phpcs:ignore Generic.CodeAnalysis.EmptyStatement.DetectedCatch
+			} catch ( InvalidCarrierException $e ) {
```

Comment:
Načtení objednávky s ignorováním výjimky už používáme hodněkrát. Co na to zavést metodu v repozitáři?

Reply:
Tato metoda by se používala jen tam, kde není pro účel prováděné akce důležité, proč se nepodařilo instanci objednávky získat. Tedy tam, kde je prázdná catch větev. Globálně ignorovat EmptyStatement.DetectedCatch mi nepřijde jako dobrý nápad.

Reply:
ok